firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -242,7 +249,16 @@ private void setUpUrlConnection(
   }
 
   private String getFetchUrl(String projectNumber, String namespace) {
-    return String.format(FETCH_REGEX_URL, projectNumber, namespace);
+    String fetchRegexUrl = FETCH_REGEX_URL;
+    if (emulatedServiceSettings != null) {
+      fetchRegexUrl =
+          "http://"
+              + emulatedServiceSettings.getHost()
```

Comment:
Will this handle IPv6 addresses correctly, like bracket-quoting (`[::1]`)?

Reply:
Yes: [android docs](https://developer.android.com/reference/java/net/HttpURLConnection#ipv6-support).